client/src/js/users/components/__tests__/PrimaryGroup.test.js

```diff
@@ -0,0 +1,47 @@
+import { mapStateToProps, mapDispatchToProps, PrimaryGroup } from "../PrimaryGroup";
+
+describe("<PrimaryGroup />", () => {
+    let props;
+    beforeEach(() => {
+        props = {
+            onSetPrimaryGroup: jest.fn(),
+            detail: { id: "foo" },
+            groups: { foo: "bar" },
+            primary_group: "fee"
+        };
+    });
+
+    it("should render", () => {
+        const wrapper = shallow(<PrimaryGroup {...props} />);
+        expect(wrapper).toMatchSnapshot();
+    });
+});
+
+describe("mapStateToProps", () => {
+    const state = {
+        users: {
+            detail: { group: "foo", primary_group: "bar" }
+        }
+    };
+    it("should return props", () => {
+        const result = mapStateToProps(state);
+        expect(result).toEqual({
+            group: "foo",
+            primary_group: "bar"
+        });
+    });
+});
+describe("mapDispatchToProps", () => {
+    it("should return editUser in props ", () => {
+        let dispatch = jest.fn();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['dispatch' is never reassigned. Use 'const' instead. (prefer-const)](https://app.codacy.com/app/virtool/virtool/pullRequest?prid=4661579)